test/GNNGraphs/transform.jl

```diff
@@ -101,6 +101,16 @@ end
     @test nodemap == 1:(g1.num_nodes)
 end
 
+@testset "remove_edges" begin
+    if GRAPH_T == :coo
+        s = [1, 1, 2, 3]
+        t = [2, 3, 4, 5]
+        g = GNNGraph(s, t, graph_type = GRAPH_T)    
+        gnew = remove_edges(g, [1])
+        @test gnew.num_edges == 3
```

Comment:
let's test that the remaining graph has the correct edges.

Reply:
done